### ISSUE TITLE
H2 to Id relation issue

### ISSUE THREAD

**Issue by fullyslick** [udacity/course-collaboration-travel-plans#1580]
In [index.html](https://github.com/udacity/course-collaboration-travel-plans/blob/master/index.html) file there is difference between the `id` of   `<div class="destination"` and the `h2 `inner html. Check out **line 13** and **line 14**, as well as l**ine 17** and **line 18**.

**Comment by nedjarsaber** [comment]
Yes, you must change the h2 content of "id belize" and "id barcelona"
